lib/cloud_controller/opi/apps_client.rb

```diff
@@ -104,10 +104,35 @@ def to_hash
       end
     end
 
+    class KpackLifecycle
+      CNB_LAUNCHER_PATH = '/cnb/lifecycle/launcher'.freeze
+
+      def initialize(process)
+        @process = process
+      end
+
+      def to_hash
+        command = if @process.command.presence
```

Comment:
Does anyone know why we use `@process.command` here instead of `@process.started_command`?

https://github.com/cloudfoundry/cloud_controller_ng/blob/bd3d96ca9a959936c6f655049e155b3edc7a7c8b/lib/cloud_controller/diego/buildpack/lifecycle_protocol.rb#L74

I'm having trouble understanding how this picks of start commands for other processes that are defined in the `Procfile`.

Reply:
Heh when I switch to `process.started_command` it doesn't work cause we run into a new bug. I don't think our build reconciler is copying over detected start commands correctly.

https://github.com/cloudfoundry/capi-k8s-release/issues/88

Reply:
I put out a PR to fix the detected start commands we're saving on droplets:
https://github.com/cloudfoundry/capi-k8s-release/pull/90

Once that is merged I can revert the "docker" -> "kpack" migration and switch this over to using `@process.started_command`.

Reply:
That's merged, so I switched this over:
https://github.com/cloudfoundry/cloud_controller_ng/pull/1917/commits/0a55501093d6a10472274f8bc134dbf4bbe60f9f